### PR TITLE
Add EXILED auto installer

### PR DESCRIPTION
**Pull request by ox-ox-ox**
I have not tested it on Windows as I don't have access to a Windows computer.

### CODE REVIEW

**Comment by xRoier** [comment]
LGTM

File: SecretAdmin/Features/Program/ProgramIntroduction.cs [hunk 1]
```diff
@@ -31,6 +31,11 @@ public static void ShowIntroduction()
 
             // Server Options
             
+            Log.Alert("Do you want to auto-install EXILED? (y/n)");
+            var ai = System.Console.ReadLine()?.ToLower();
+            if (ai != null && (ai[0] == 'y' || ai[0] == 'n'))
```
Comment (TheUltiOne) [inline SecretAdmin/Features/Program/ProgramIntroduction.cs:36]: ai[0] == 'n', and installs Exiled?
Am I not seeing something

**Review by TheUltiOne — COMMENTED** [review]
I’d also recommend using a dictionary for answers.

File: SecretAdmin/Features/Program/ProgramIntroduction.cs (hunk 1 above)
Comment (ox-ox-ox) [reply SecretAdmin/Features/Program/ProgramIntroduction.cs:36]: I may have pulled a smoothbrain 😔 
Jesus fixed this in a later commit

**Pull request merged**